src/main/java/Storage.java

```diff
@@ -0,0 +1,87 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    public String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+
+    /**
+     * Retrieves tasks from Hard Disk.
+     * @param path Path to file in Hard Disk.
```

Comment:
Perhaps add a blank line in Line 18? To separate the docs from @param as you've did for the above few files!